HW_1/task_1.py

```diff
@@ -0,0 +1,23 @@
+"""
+Задача 2: Найдите сумму цифр трехзначного числа.
+*Пример:*
+123 -> 6 (1 + 2 + 3)
+100 -> 1 (1 + 0 + 0)
+"""
+
+print('Введите трехзначное число: ')
+m = int(input())
```

Comment:
Зачем вы использовали функцию `print` для вывода сообщения перед вводом, когда это самое сообщение можно использовать прямо в функции `input`
```suggestion
m = int(input('Введите трехзначное число: '))
```